slack.go

```diff
@@ -54,6 +63,20 @@ func listenToSlackEvents(client *socketmode.Client, api *slack.Client, config *c
 
 		switch evt.Type {
 
+		case socketmode.EventTypeEventsAPI:
+			apiEvent, ok := evt.Data.(slackevents.EventsAPIEvent)
+			if !ok {
+				logger.Errorw("could not type cast the event to the EventsAPIEvent", "event", evt)
+				continue
+			}
+
+			client.Ack(*evt.Request)
+
+			err := handleEventMessage(api, config, apiEvent)
+			if err != nil {
+				logger.Errorw("unable to handle EventsAPI event", "event", evt, "error", err)
+			}
+
```

Comment:
nit: could be nice to abstract each of the case statement logic into functions

```suggestion
		case socketmode.EventTypeEventsAPI:
                     handleEventTypeEvents()
```

Reply:
It is abstracted already, this case statement just 1) casts the event and 2) passes it to the handle func

Do we need a new handle func to then call this other handle func? 🤔